openfisca_aotearoa/variables/acts/social_security/supported_living_payment/supported_living_payment.py

```diff
@@ -1,124 +1,236 @@
+
+        ssa40D_2 = ssa40D_2_a + ssa40D_2_b
+
+        # person C meets the residential requirement
+        ssa40D_3 = persons("social_security__residential_requirement", period.first_week)
+
+        # ssa40D_4 allows for payment to continue if not caring for 28 days
+
+        return ssa40D_1 * ssa40D_2 * ssa40D_3
+
+    def formula_2016_10_25(persons, period, parameters):
+        # person C is caring for person P
+        ssa40D_1 = persons("supported_living_payment__caring_for_another_person", period.first_week)
+
+        # person C is at least 18 years old if they have no dependent children
+        ssa40D_2_a = (persons("age", period.start) >= 18) * (persons("social_security__dependent_children", period.first_week) == 0)
+
+        # person C is at least 20 years old otherwise (i.e.: they have dependent children)
+        ssa40D_2_b = persons("age", period.start) >= 20
+
+        ssa40D_2 = ssa40D_2_a + ssa40D_2_b
+
+        # person C meets the residential requirement
+        ssa40D_3 = persons("social_security__residential_requirement", period.first_week)
+
+        # ssa40D_4 allows for payment to continue if not caring for 28 days
+
+        return ssa40D_1 * ssa40D_2 * ssa40D_3
+
+    def formula_2018_11_26(persons, period, parameters):
+        # person C is caring for person P
+        ssa40_1_a = persons("supported_living_payment__caring_for_another_person", period.first_week)
+
+        # person C meets the residential requirement
+        ssa40_1_b = persons("social_security__residential_requirement", period.first_week)
+
+        # person C is at least 18 years old if they have no dependent children
+        ssa40_1_c_i = (persons("age", period.start) >= 18) * (persons("social_security__dependent_children", period.first_week) == 0)
+
+        # person C is at least 20 years old otherwise (i.e.: they have dependent children)
+        ssa40_1_c_ii = persons("age", period.start) >= 20
+
+        ssa40_1_c = ssa40_1_c_i + ssa40_1_c_ii
+
+        # ssa40_2 allows for payment to continue if not caring for 28 days
+
+        # ssa40_3 defines "institutional care"
+
+        # ssa41 requires a certificate when applying for supported living payment as a carer
+
+        # ssa42 specifies requirements for medical examination
+
+        # ssa54_5_d person C is ineligible if receiving youth payment
+
+        # ssa61_5_d person C is ineligible if receiving young parent payment
+
+        return ssa40_1_a * ssa40_1_b * ssa40_1_c
 
 
 class supported_living_payment__restricted_work_capacity(variables.Variable):
     value_type = bool
+    default_value = False
     entity = entities.Person
     definition_period = periods.WEEK
     label = "Is incapable of regularly working 15 or more hours a week in open employment"
-    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783176", "http://legislation.govt.nz/act/public/1964/0136/latest/whole.html#DLM5468367"
+    reference = [
```

Comment:
Much prefer this format, it'd be good to add this to the Contributing documentation